src/InformaticsGateway/Services/Scu/ScuService.cs

```diff
@@ -56,7 +56,7 @@ internal sealed class ScuService : IHostedService, IDisposable, IMonaiService
             _workQueue = _scope.ServiceProvider.GetService<IScuQueue>() ?? throw new ServiceNotFoundException(nameof(IScuQueue));
         }
 
-        private async Task BackgroundProcessingAsync(CancellationToken cancellationToken)
+        private Task BackgroundProcessingAsync(CancellationToken cancellationToken)
```

Comment:
@mocsharp sorry maybe a stupid quesion but should the name of this method change? or is it inherently async in nature?

Reply:
Absolutely! Thanks for catching that.